Tools/autotest/pysim/util.py

```diff
@@ -137,13 +140,14 @@ def build_SITL(build_target, j=None, debug=False, board='sitl', clean=True, conf
     return True
 
 
-def build_examples(board, j=None, debug=False, clean=False, configure=True, math_check_indexes=False, extra_configure_args=[]):
+def build_examples(board, j=None, debug=False, clean=False, configure=True, math_check_indexes=False, coverage=False, extra_configure_args=[]):
     # first configure
     if configure:
         waf_configure(board,
                       j=j,
                       debug=debug,
                       math_check_indexes=math_check_indexes,
+                      coverage=coverage,
```

Comment:
This is what I was saying about making an opaque object for arguments to configure; you've had to make 6 extra patches in this PR which you wouldn't have had to!